src/components/client-registration.tsx

```diff
@@ -96,7 +96,7 @@ const  ClientRegistration: React.FC<{config: AuthConfig, updateRegistrationSucce
     const [resourceOwner, setResourceOwner] = useState("");
     const [resourceOwnerEmail, setResourceOwnerEmail] = useState("");
     const [additionalInfo, setAdditionalInfo] = useState("");
-    const [scopes, setScopes] = useState("openid email profile roles"); 
+    const [scopes, setScopes] = useState("openid email profile ala/roles"); 
```

Comment:
This change needs to be done once the cognito is released? Until that token generation happens with CAS.